include/ebpf_structs.h

```diff
@@ -408,3 +413,12 @@ struct bpf_prog_info
     uint32_t pinned_path_count;          ///< Number of pinned paths.
     uint32_t link_count;                 ///< Number of attached links.
 };
+
+/* BPF_FUNC_perf_event_output flags. */
+#define EBPF_MAP_FLAG_INDEX_MASK 0xffffffffULL
+#define EBPF_MAP_FLAG_INDEX_SHIFT 0
+#define EBPF_MAP_FLAG_CURRENT_CPU EBPF_MAP_FLAG_INDEX_MASK
+/* BPF_FUNC_perf_event_output flags for program types with data pointer in context */
```

Comment:
```suggestion
/* BPF_FUNC_perf_event_output flags for program types with data pointer in context. */
```
nit: For consistency with line 417